### PR TITLE
Dm2 fu 01

### PR DESCRIPTION
**Pull request by ralflukner**
Fixes # Not A Fix

#### Short description of what this resolves:
Not a fix

#### Changes proposed in this pull request:
Forms for Diabetes Mellitus Type 2 Follow-up Visit
dm2_fu_hpi_01 - hpi form
dm2_fu_ap_01 - a/p form
dm2_fu_labs_01 - labs form

below added by bradymiller:
Up For Grabs demo for this PR is at:
https://www.open-emr.org/wiki/index.php/Development_Demo#Gamma_-_Up_For_Grabs_Demo

### CODE REVIEW

**Comment by stephenwaite** [comment]
nice, that's more like it :)
you can add back `.gitignore` since it's a valid file in the repo

**Comment by ralflukner** [comment]
Okay.  I put openemr/.idea/.gitignore back.

**Comment by stephenwaite** [comment]
thanks @ralflukner , we'll check it out this weekend, thanks for the contribution

**Comment by stephenwaite** [comment]
hi @ralflukner , how is this testing for you?
Just a quick look over but seems like all the files related to your new form should either belong under the `interface/forms` directory or `contrib/forms` like how you've emulated the `interface/forms/soap` form for instance.

**Comment by bradymiller** [comment]
hi, Would put them in `interface/forms`. Then I can set up a Up For Grabs demo on this PR for easy testing.

**Comment by ralflukner** [comment]
moved new forms to the interface/forms directory for easy testing.

**Comment by sjpadgett** [comment]
Really nice to watch your progress @ralflukner. I firmly believe that health care professionals make great contributor to a project such as ours. Thanks for the contribution.

**Comment by bradymiller** [comment]
Up For Grabs demo for this PR is at:
https://www.open-emr.org/wiki/index.php/Development_Demo#Gamma_-_Up_For_Grabs_Demo
(note this demo resets daily at either midnight or 1 am pacific time and then builds from most recent code in this PR)

**Comment by ralflukner** [comment]
> Up For Grabs demo for this PR is at: https://www.open-emr.org/wiki/index.php/Development_Demo#Gamma_-_Up_For_Grabs_Demo (note this demo resets daily at either midnight or 1 am pacific time and then builds from most recent code in this PR)

It seems to be working as far as I can tell...  I tested all three forms _hpi_, _labs_, and _ap_

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```
Comment (stephenwaite) [inline .gitignore:8]: hi @ralflukner , these seem like they are specific to only your setup and so should probably be removed.

File: .gitignore [hunk 2]
```diff
@@ -26,3 +34,5 @@ ccdaservice/node_modules
 
 # testing
 .phpunit.result.cache
+.idea/workspace.xml
+.idea/workspace.xml
```
Comment (stephenwaite) [inline .gitignore:38]: the `/.idea/*` above should capture any related files so would remove these as well

File: interface/forms/dm2_fu_ap_01/info.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+DM2_FU_AP_01
```
Comment (stephenwaite) [inline interface/forms/dm2_fu_ap_01/info.txt:1]: would make the display name more human readable

Screenshot from 2021-11-23 06-52-45

**Review by stephenwaite — COMMENTED** [review]
nice job emulating an existing form and adding these in, just noted some minor items

File: interface/forms/dm2_fu_ap_01/info.txt (hunk 3 above)
Comment (bradymiller) [reply interface/forms/dm2_fu_ap_01/info.txt:1]: Agree with @stephenwaite that the form names should be human readable (ie. what is in the info.txt files). When that is addressed, then this PR can be brought into the codebase.

**Pull request closed without merging**